src/jio-navbar.css

```diff
@@ -4,8 +4,7 @@ a:focus-visible {
 
 .navbar-toggler {
   background-color: transparent;
-  border: 1px solid transparent;
-  border-color: rgba(255, 255, 255, 0.1);
+  border: 1px solid rgba(255, 255, 255, 0.1);
```

Comment:
what does "Fixed some src/jio-navbar.css CSS styling issues." actually do?
do you have a screenshot or something?
maybe a seperate pr or issue so the changelog makes sense?

Reply:
It is just because there were some duplicate CSS stylings and I could see some errors cropping up in my IDE

Reply:
mmmm, makes sense. I would have said "removed warning about duplicate css entries" or something as its not really a fix.
Either way, i guess it can be grouped into this one if the other point gets approved, or split into two prs so this gets merged no matter what.